internal/transport/handler_server_test.go

```diff
@@ -308,16 +336,15 @@ func handleStreamCloseBodyTest(t *testing.T, statusCode codes.Code, msg string)
 		func(ctx context.Context, method string) context.Context { return ctx },
 	)
 	wantHeader := http.Header{
-		"Date":         nil,
+		"Date":         {},
 		"Content-Type": {"application/grpc"},
 		"Trailer":      {"Grpc-Status", "Grpc-Message", "Grpc-Status-Details-Bin"},
+	}
+	wantTrailer := http.Header{
```

Comment:
I guess this (and a test or two below) will have to be updated when #553 is eventually fixed, as there *should* only be one headers frame sent for these kinds of RPCs.  (This is purely FYI.)